bryft/src/pages/AboutPage/AboutPage.js

```diff
@@ -0,0 +1,58 @@
+import "./AboutPage.css";
+import Navbar from "../../components/Navbar/Navbar";
+import IconButton from "@mui/material/IconButton";
+import Email from "@mui/icons-material/Email";
+import Instagram from "@mui/icons-material/Instagram";
+import LinkedIn from "@mui/icons-material/LinkedIn";
+import { Link } from 'react-router-dom';
+
+import alan from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/alanheadshot.jpeg';
+import kenny from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/kennyheadshot.jpg';
+import kevin from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/kevinheadshot.jpeg';
+import mikey from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/mikeyheadshot.jpeg';
+import sophia from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/sophiaheadshot.jpg';
+
+
+function AboutPage() {
+    return (
+        <div>
+        <Navbar />
+        <div className="brick" style = {{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
```

Comment:
Can we put these styles a css sheet? It'll make the code look a lot cleaner + we should avoid mixing css and html together for industry practice anyway cause most of these styles are the same.